cmd/dcrdata/views/market.tmpl

```diff
@@ -18,11 +18,11 @@
 
         {{- /* PRICE */ -}}
         <div class="ms-4 me-2 my-4 p-2 p-lg-4 bg-white text-center">
-            <div class="text-center fs18 text-secondary">1 DCR =</div>
+            <span class="text-center fs18 text-secondary">1 DCR =</span>
             {{if eq $botState.BtcIndex "USD"}}
-                <span class="fs18 fw-bold d-inline-block pt-1" style="vertical-align:top;">$</span>
```

Comment:
Not sure about this.  I certainly wouldn't call the dollar sign formatting a "quirk", since it's an intentional stylization, which both allows the dollar sign to be smaller (it looks huge with this PR), as well as a bit fancy.

![image](https://user-images.githubusercontent.com/9373513/205951878-dfc75057-89f9-47fb-9ec1-0d0e5a022d2d.png)

But again, if others think this style is more odd than pleasant, we can change it.  Others with opinions on these?

Reply:
While it looks intentional, I think it is more popular to this display the dollar sign right beside the figure using "almost" the same size?

Reply:
Fine with me either way.  Opinions @buck54321 @xaur @jzbz @davecgh 
Big dollar sign or funky stylization we have presently?

Reply:
I'm okay with both as long as I can select `$ 16,966.68` and copy it into clipboard (i.e. it is plain ASCII and not a fancy icon or an exotic Unicode char). I slightly prefer `$` of the "almost" same size/font because my eyes got used to it.

While at it, adding a thousand separator like `1,000,000` to any number that can go above 1,000 would help readability.

Reply:
I'm fine with either.  If I absolutely had to choose, I find the currency symbol being the same size as the digits to more aesthetically pleasing.  It's probably just because that's how I'm more used to seeing it though and it really doesn't bother me stylized either.

Reply:
> While at it, adding a thousand separator like `1,000,000` to any number that can go above 1,000 would help readability.

Good suggestion.  Can probably use `humanize.CommafWithDigits` in a new entry in the `template.FuncMap`.

Reply:
Oh right.  toLocaleString with 2 for min and max decimals should work.